pacman/model/graphs/application/application_vertex.py

```diff
@@ -58,35 +68,30 @@ def __repr__(self):
         return "ApplicationVertex(label={}, constraints={}".format(
             self.label, self.constraints)
 
-    @abstractmethod
-    def get_resources_used_by_atoms(self, vertex_slice):
-        """ Get the separate resource requirements for a range of atoms
+    @property
+    def splitter_object(self):
+        return self._splitter_object
```

Comment:
Only the getter documentation appears in Sphinx's output. No documentation on the getter, no documentation on the property. All that we really need is:
```python
"""
:rtype: ~pacman.model.partitioner_interfaces.AbstractSplitterCommon
"""
```

Reply:
fixed